client/js/shout.js

```diff
@@ -15,6 +15,7 @@ $(function() {
 		"/nick",
 		"/notice",
 		"/op",
+		"/otr",
```

Comment:
I am wondering if this should not be `"/otr init"` instead, but I'm not sure how the UI reacts to auto-complete with spaces (I haven't tried).